packages/cozy-client/src/CozyClient.js

```diff
@@ -1236,7 +1239,14 @@ client.query(Q('io.cozy.bills'))`)
   register(cozyURL) {
     const stackClient = this.getStackClient()
     stackClient.setUri(cozyURL)
-    return this.startOAuthFlow(authenticateWithCordova)
+    const authFunction = this.isReactNative
+      ? authenticateWithReactNativeInAppBrowser
+      : authenticateWithCordova
+    return this.startOAuthFlow(authFunction)
+  }
+
+  isReactNative() {
```

Comment:
I think we should move the method to `src/authentication/mobile.js` 

Also, I think it more readable and extendable to have a `getAuthenticationMethod` in authent/mobile.js , and use it in `register` and `renew`. 

Like that, if we have to add others methods, it'll be easier. 

(also, `isReactNative` should not be part of the client API)